logbook-servlet/src/main/java/org/zalando/logbook/servlet/RemoteRequest.java

```diff
@@ -166,20 +146,24 @@ public byte[] getBody() {
             return body;
         }
 
+        @Override
+        public ServletInputStream getInputStream(final ServletRequest request) {
+            return new ServletInputStreamAdapter(new ByteArrayInputStream(body));
```

Comment:
Using `new ByteArrayInputStream(body)` can be a slow operation, especially for a large `body`. Is resolving this bug (which seems a rather edge case) worth doing it always?

An alternative might be giving the user an option to deep-clone the body, or simply return it.